certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,688 @@
+                      },
+                      "Cluster_Label": {
+                        "type": "string",
+                        "description": "Cluster_Label"
+                      }
+                    }
+                  },
+                  "description": "Results"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create text analytic job",
+        "description": "Submit text analytic job to Business Assist.",
+        "consumes": ["application/json"],
+        "operationId": "CreateCluserting",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Name": {
+                  "type": "string",
+                  "description": "Name"
+                },
+                "InputData": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "RecordId": {
+                        "type": "string",
+                        "description": "RecordId"
+                      },
+                      "Text": {
+                        "type": "string",
+                        "description": "Text"
+                      }
+                    }
+                  },
+                  "description": "InputData"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/api/beta/textanalytics/topicclustering/{Clustering job Id}/status": {
+      "get": {
+        "responses": {
+          "202": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "OperationId": {
+                  "type": "string",
+                  "description": "Clustering operation Id"
+                },
+                "Name": {
+                  "type": "string",
+                  "description": "Clustering Name"
+                },
+                "Status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Clustering Status"
+                },
+                "Diagnostics": {
+                  "type": "string",
+                  "description": "Diagnostics message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get text analytic job status",
+        "description": "Get text analytic job status by job id.",
+        "operationId": "GetClusteringStatus",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Clustering job Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "a6fe7af6-ba42-4676-9a4c-901d0bd44d19",
```

Comment:
@sjagaleti please change the value of the description. Secret look alike values are not allowed.